src/features/chat/ui/chatRoomList/index.tsx

```
@@ -0,0 +1,63 @@
+    setFilteredRoom(rooms);
+  }, [rooms]);
+
+  return (
+    <section className="h-full w-full flex-1/5 border-r border-border_primary/20">
+      <div className="flex items-center justify-between p-4 border-b border-border_primary/20">
+        <h2 className="text-xl font-semibold text-white">채팅</h2>
+        <button
+          onClick={closeChatDialog}
+          className="hover:bg-white/10 rounded-[10px] transition-colors"
+        >
+          <Image src="/icons/close/closeWithBorder.svg" height={20} width={20} alt="closeBtn" />
```

Biome (2.1.2)

[error] Provide an explicit type prop for the button element.

The default type of a button is submit, which causes the submission of a form when placed inside a `form` element. This is likely not the behaviour that you want inside a React application.
Allowed button types are: submit, button or reset

(lint/a11y/useButtonType)